test/new_relic/agent/instrumentation/rails/active_storage_subscriber.rb

```diff
@@ -1,6 +1,7 @@
 # encoding: utf-8
 # This file is distributed under New Relic's license terms.
 # See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+require File.expand_path '../../../../../test_helper', __FILE__
```

Comment:
👍 This is one of the better implementations of this `require` that would make a good candidate for https://github.com/newrelic/newrelic-ruby-agent/issues/1094